secure-api-gateway-ob-uk-common-datamodel/src/main/java/com/forgerock/sapi/gateway/ob/uk/common/datamodel/converter/v4/event/FREventPollingConverter.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2020-2024 ForgeRock AS (obst@forgerock.com)
```

Comment:
Just 2024 if these are both new?
(Also in other class)

Reply:
This is the mandatory format of copyright header

Reply:
2020 though? For a new file... Should it not just be 2024?

Reply:
this is the template in the pipeline at copyright check stage